v2/samples/synapse/v1beta/v1beta20210601_synapsebigdatapool.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: synapse.azure.com/v1beta20210601
+kind: WorkspacesBigDataPool
+metadata:
+  name: samplepool
+  namespace: default
+spec:
+  location: westcentralus
+  owner:
```

Comment:
These samples need to have a recording made for them. You can do this by running the "Test_Samples_CreationAndDeletion" test. I've sent you a PR though with this already done so you don't have to wrestle through it this time.